### PR TITLE
Bootstrap

### PR DESCRIPTION
**Pull request by isidzukuri**

### CODE REVIEW

File: app/views/categories/_form.html.haml [hunk 1]
```diff
@@ -1,15 +1,27 @@
-= form_for @item do |f|
-  %p
-    = f.label :title
+.panel-body
+  = form_for @item do |f|
+    .input-group
+      = f.label :title
+      %br/
+      = f.text_field :title, :class => 'form-control'
```
Comment (AKovtunov) [inline app/views/categories/_form.html.haml]: different hash styles here

File: app/views/categories/_form.html.haml [hunk 2]
```diff
@@ -1,15 +1,27 @@
...
-  %p
-    = f.submit
-
+    .input-group
+      = f.label :parent
+      %br/
+      = f.select :parent_id, options_from_collection_for_select(Category.all, :id, :title, @item.parent_id), {:include_blank => true}, {:class => "form-control"}
+    %br/
+    .input-group
+      = f.submit class: 'btn btn-primary'
```
Comment (AKovtunov) [inline app/views/categories/_form.html.haml]: and here
it's not ok, use same ones (better new-style hash)

File: app/views/categories/_form.html.haml (hunk 1 above)
Comment (isidzukuri) [inline app/views/categories/_form.html.haml]: fixed

File: app/views/categories/_form.html.haml (hunk 2 above)
Comment (isidzukuri) [inline app/views/categories/_form.html.haml]: fixed

**Pull request merged**